apps/prairielearn/src/lib/externalGradingSocket.js

```diff
@@ -127,34 +129,6 @@ module.exports.gradingJobStatusUpdated = function (grading_job_id) {
   });
 };
 
-module.exports.renderPanelsForSubmission = function (
-  submission_id,
-  question_id,
-  instance_question_id,
-  variant_id,
-  urlPrefix,
-  questionContext,
-  csrfToken,
-  authorizedEdit,
-  callback,
-) {
-  question.renderPanelsForSubmission(
-    submission_id,
-    question_id,
-    instance_question_id,
-    variant_id,
-    urlPrefix,
-    questionContext,
-    csrfToken,
-    authorizedEdit,
-    true, // renderScorePanels
-    (err, results) => {
-      if (ERR(err, callback)) return;
-      callback(null, results);
-    },
-  );
-};
-
```

Comment:
This function was not used anywhere else in the code, so I changed it's only call to go directly to the lib.